doc/design/test/nsx.md

```diff
@@ -0,0 +1,49 @@
+# NSX Initial testing notes
+
+##Required HW setup:
+- vSphere 6.0 Cluster setup
```

Comment:
Apparently, they just released a new version that support 6.5 finally - so just make a note of that here (that 6.5 wasn't supported for awhile, but latest releases now support)